src/containers/PageBuilder/SectionBuilder/SectionFooter/SectionFooter.module.css

```diff
@@ -0,0 +1,185 @@
+@import '../../../../styles/customMediaQueries.css';
+
+.footer {
+  /* padding: 32px;
+  max-width: calc(var(--contentMaxWidth) + 72px);
+  margin: 0 auto; */
+  max-width: var(--contentMaxWidthPages);
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 32px;
+  margin: 0 auto;
+  padding: 32px;
+}
+
+.footer ul {
+  list-style-type: none;
+  padding-inline-start: 0px;
+}
+
+.linkList {
+  list-style-type: none;
```

Comment:
Add something about Footer to SectionBuilder/README.md

There are some special rules like this one. I.e. "list-item" doesn't look like a default bullet point list, because the expected use case is to create a list of links with that particular markdown syntax.

Reply:
Have you tested other markdown syntax in the footer - does it look OK?

Reply:
I tested markdown syntax in the footer, and bolding, italics and block quotes work. Heading levels work apart from font size, i.e. headings get bolding and added margin but stay the same size.

How detailed should the README.md explanation be on SectionFooter?